<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButtonIncluirInicio">
    <property name="geometry">
     <rect>
      <x>189</x>
      <y>90</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Incluir Inicio</string>
    </property>
    <property name="icon">
     <iconset resource="Arquivos.qrc">
      <normaloff>:/Imagens/Arquivos/inserir.png</normaloff>:/Imagens/Arquivos/inserir.png</iconset>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>220</y>
      <width>47</width>
      <height>39</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:700;&quot;&gt;Valor&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonAcessarFinal">
    <property name="geometry">
     <rect>
      <x>337</x>
      <y>120</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Acessar Final</string>
    </property>
    <property name="icon">
     <iconset resource="Arquivos.qrc">
      <normaloff>:/Imagens/Arquivos/acessar.png</normaloff>:/Imagens/Arquivos/acessar.png</iconset>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>60</y>
      <width>61</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:700;&quot;&gt;Incluir&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>60</y>
      <width>61</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:700;&quot;&gt;Retirar&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>310</y>
      <width>121</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:700;&quot;&gt;Mostrar Lista&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>60</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:700;&quot;&gt;Acessar&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonIncluirFinal">
    <property name="geometry">
     <rect>
      <x>189</x>
      <y>120</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Incluir Final</string>
    </property>
    <property name="icon">
     <iconset resource="Arquivos.qrc">
      <normaloff>:/Imagens/Arquivos/inserir.png</normaloff>:/Imagens/Arquivos/inserir.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonRetirarInicio">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>90</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Retirar Início</string>
    </property>
    <property name="icon">
     <iconset resource="Arquivos.qrc">
      <normaloff>:/Imagens/Arquivos/retirar.png</normaloff>:/Imagens/Arquivos/retirar.png</iconset>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditSaida">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>340</y>
      <width>569</width>
      <height>159</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonAcessarInicio">
    <property name="geometry">
     <rect>
      <x>337</x>
      <y>90</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Acessar Início</string>
    </property>
    <property name="icon">
     <iconset resource="Arquivos.qrc">
      <normaloff>:/Imagens/Arquivos/acessar.png</normaloff>:/Imagens/Arquivos/acessar.png</iconset>
    </property>
   </widget>
   <widget class="QLabel" name="labelFoto">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>70</y>
      <width>81</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="Arquivos.qrc">:/Imagens/Arquivos/jhone.jpeg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonRetirarFinal">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>120</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Retirar Final</string>
    </property>
    <property name="icon">
     <iconset resource="Arquivos.qrc">
      <normaloff>:/Imagens/Arquivos/retirar.png</normaloff>:/Imagens/Arquivos/retirar.png</iconset>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditValor">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>230</y>
      <width>169</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>0</y>
      <width>221</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; font-weight:700; font-style:italic; color:#000000;&quot;&gt;Sistema Teste Lista&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>280</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:700;&quot;&gt;Quantidade de Elementos &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelFundo">
    <property name="geometry">
     <rect>
      <x>-2</x>
      <y>-25</y>
      <width>811</width>
      <height>611</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="Arquivos.qrc">:/Imagens/Arquivos/windows.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditQuantidadeElementos">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>280</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <zorder>labelFundo</zorder>
   <zorder>pushButtonIncluirInicio</zorder>
   <zorder>label_5</zorder>
   <zorder>pushButtonAcessarFinal</zorder>
   <zorder>label_2</zorder>
   <zorder>label_4</zorder>
   <zorder>label_6</zorder>
   <zorder>label_3</zorder>
   <zorder>pushButtonIncluirFinal</zorder>
   <zorder>pushButtonRetirarInicio</zorder>
   <zorder>textEditSaida</zorder>
   <zorder>pushButtonAcessarInicio</zorder>
   <zorder>pushButtonRetirarFinal</zorder>
   <zorder>lineEditValor</zorder>
   <zorder>label</zorder>
   <zorder>label_7</zorder>
   <zorder>textEditQuantidadeElementos</zorder>
   <zorder>labelFoto</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="Arquivos.qrc"/>
 </resources>
 <connections/>
</ui>
